packages/conf-gcc-multilib/conf-gcc-multilib.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "Simmo Saan <simmo.saan@gmail.com>"
+authors: ["Simmo Saan"]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-2.0-or-later"
+build: [
+  ["sh" "-exc" "echo '#include <limits.h>' | cpp -m32 -"]
+  ["sh" "-exc" "echo '#include <limits.h>' | cpp -m64 -"]
+]
+depexts: [
+  ["glibc-devel.i686"] {os-distribution = "centos"}
+  ["glibc-devel.i686"] {os-distribution = "fedora"}
+  ["gcc-32bit"] {os-family = "suse" | os-distribution = "opensuse"}
+  ["gcc-multilib"] {os-distribution = "debian"}
+  ["gcc-multilib"] {os-distribution = "ubuntu"}
+  ["lib32-glibc"] {os-distribution = "archlinux"}
```

Comment:
```suggestion
  ["lib32-glibc"] {os-family = "arch"}
```

See https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values

Also, orthogonally, I wonder if this should be `lib32-gcc-libs`